Parse/PFFile.m

```diff
@@ -315,13 +307,13 @@ - (BFTask *)_uploadFileAsyncWithSessionToken:(NSString *)sessionToken
 #pragma mark Download
 
 - (BFTask *)_getDataAsyncWithProgressBlock:(PFProgressBlock)progressBlock {
-    return [[self _downloadAsyncWithProgressBlock:progressBlock] continueAsyncWithSuccessBlock:^id(BFTask *task) {
+    return [[self _downloadAsyncWithProgressBlock:progressBlock] continueWithSuccessBlock:^id(BFTask *task) {
```

Comment:
Why was this async before? Wasn't it to ensure that all the progress block ticks finished?